scripts/qesap/lib/cmds.py

```diff
@@ -345,18 +349,15 @@ def cmd_ansible(configure_data, base_project, dryrun, verbose, destroy=False, pr
         return Status("ok")
 
     inventory = os.path.join(base_project, 'terraform', configure_data['provider'], 'inventory.yaml')
-    ansible_cmd_seq = ansible_command_sequence(configure_data['ansible'], base_project, sequence, verbose, inventory, profile)
+    ansible_cmd_seq = ansible_command_sequence(configure_data['ansible'], base_project, sequence, False, inventory, profile)
```

Comment:
Removed the verbose output, not sure if it impacts json output in any way, I think not

Reply:
Maybe not but will afect this pr already merge https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18787